server/dates.ts

```diff
@@ -30,7 +30,7 @@ export const postDate = async (
     return res.status(401).send({ message: 'User is not an admin' });
   }
 
-  const newDate: string = JSON.stringify(new Date(req.body.date));
+  const newDate: string = new Date(req.body.date).toISOString();
```

Comment:
`toISOString` converts dates to GMT. On the portal, we want the dates to be set as EST.